src/Couchbase.Lite.Shared/Document/DataOps.cs

```diff
@@ -110,6 +110,8 @@ internal static object ToCouchbaseObject(object value)
                     return null;
                 case DateTimeOffset dto:
                     return dto.ToString("o");
+                case DateTime dt: //json serialized datetimeoffset to date time so convert it dto
+                    return new DateTimeOffset(dt.ToUniversalTime());
```

Comment:
There is a setting to control this, you should make use of it like other classes in Couchbase Lite

Reply:
I still notice that you are handling `DateTime` here when we didn't before.  Everything should be using `DateTimeOffset`